Sources/Purchasing/Purchases/PurchasesOrchestrator.swift

```diff
@@ -593,7 +606,11 @@ final class PurchasesOrchestrator {
 
             self.cachePresentedOfferingContext(package: package, productIdentifier: sk2Product.id)
 
-            result = try await self.purchase(sk2Product, options)
+            result = try await self.storeKit2ProductPurchaser.purchase(
```

Comment:
This function has the exact same function signature as the old purchase() function, except for the addition of the `storeKit2ConfirmInOptions`